src/config/oranda.rs

```diff
@@ -38,16 +35,13 @@ pub struct OrandaConfig {
 }
 
 impl OrandaConfig {
-    pub fn load() -> Result<Option<OrandaConfig>> {
+    pub fn load(config_path: &Path) -> Result<Option<OrandaConfig>> {
         println!("reading from oranda config...");
-        if Path::new(ORANDA_JSON).exists() {
-            let oranda_json = fs::read_to_string(ORANDA_JSON)?;
-            println!("read json: {:?}", &oranda_json);
-            let data: OrandaConfig = serde_json::from_str(&oranda_json)?;
-            println!("read data: {:?}", &data);
-            Ok(Some(data))
-        } else {
-            Ok(None)
-        }
+        let config_future = axoasset::load_string(config_path.to_str().unwrap());
+
+        let config = tokio::runtime::Handle::current().block_on(config_future)?;
+        let data: OrandaConfig = serde_json::from_str(config.as_str())?;
+        println!("read data: {:?}", &data);
```

Comment:
did you mean to keep this?

Reply:
deleted this and also the ones from the js and rust config files